### ISSUE TITLE
Prefixing commands with command makes it easy to create stubs

### ISSUE THREAD

**Issue by alexharv074** [kward/shunit2#54]
I have been using this project and "stubbing" out system commands and shell built-ins by defining functions in my tests.

It would be nice if I could also stub out built-ins/commands that are also used by shUnit2 internally, e.g."

``` bash
[() {
  case "$*" in
  "-x /usr/local/rvm/bin/rvm ]")
    false
    ;;
  esac
}
```

Unfortunately, this currently breaks shUnit2 itself.

As mentioned [here](https://stackoverflow.com/questions/44870065/stubbing-file-tests-in-shunit2/44870101), however, this problem could be avoided if all calls to `[` and other commands that the user might want to stub, like `mkdir`, `rm` etc, were prefixed by `command`.

E.g. instead of (on line 17)

``` bash
[ -n "${SHUNIT_VERSION:-}" ] && exit 0
```

we would have

``` bash
\[ -n "${SHUNIT_VERSION:-}" ] && exit 0
```

I tested this - and applied to a few other commands that a user might want to stub, like rm, mkdir, cat etc.

**Comment by kward** [comment]
Pull request committed. Tks!

**Comment by alexharv074** [comment]
This person found a different way to stub commands https://github.com/zofrex/bourne-shell-unit-testing  This is useful, because this feature will probably have to be reverted.

**Comment by kward** [comment]
This solution is causing quite a few problems under macOS Catalina and the /bin/dash shell. I'm looking for some alternatives that meet your needs.

**Comment by kward** [comment]
Fixed in 86f430c.
